### PR TITLE
feat: Display which player is which car

### PR DESCRIPTION
**Pull request by RatikKapoor**
image

image

### CODE REVIEW

File: DustRiders/DustRiders/Overlay.cpp [hunk 1]
```diff
@@ -374,6 +374,26 @@ void Overlay::RenderPause(StateHandler::GameState prevState, int windowHeight, i
 	ImGui::Text("ESC or START to toggle pause");
 	ImGui::End();
 
+	ImGui::SetNextWindowPos(ImVec2(windowWidth * 1.6, windowHeight * 0.6));
```
Comment (RatikKapoor) [inline DustRiders/DustRiders/Overlay.cpp:377]: @brooke-k can you confirm this looks good on your 1080p screen

File: DustRiders/DustRiders/Overlay.cpp [hunk 2]
```diff
@@ -421,6 +441,26 @@ void Overlay::RenderMenu(int windowHeight, int windowWidth)
 	ImGui::Text("ESC or START to toggle pause");
 	ImGui::End();
 
+	ImGui::SetNextWindowPos(ImVec2(windowWidth * 1.6, windowHeight * 1.6));
```
Comment (RatikKapoor) [inline DustRiders/DustRiders/Overlay.cpp:444]: @brooke-k same confirm here

File: DustRiders/DustRiders/Overlay.cpp (hunk 1 above)
Comment (brooke-k) [reply DustRiders/DustRiders/Overlay.cpp:377]: Pushed a commit moving the text up slightly so it isn't cut off at the bottom of the screen (a one character change)

File: DustRiders/DustRiders/Overlay.cpp (hunk 2 above)
Comment (brooke-k) [reply DustRiders/DustRiders/Overlay.cpp:444]: Confirmed

**Review by brooke-k — APPROVED** [review]
Looks great!

**Pull request merged**